crypto/objects/obj_dat.c

```diff
@@ -729,17 +729,28 @@ int OBJ_create(const char *oid, const char *sn, const char *ln)
     ASN1_OBJECT *tmpoid = NULL;
     int ok = 0;
 
+    /* With not arguments at all, nothing can be done */
+    if (oid == NULL && sn == NULL && ln == NULL) {
+        ERR_raise(ERR_LIB_OBJ, ERR_R_PASSED_INVALID_ARGUMENT);
```

Comment:
`ERR_R_PASSED_NULL_PARAMETER` ?

Reply:
That's going to be confusing...  'cause you *are* allowed to pass NULL, just not on all at once.  So this particular *combination* is invalid.